### ISSUE TITLE
mdList secondary button - Aria Label [small bug]

### ISSUE THREAD

**Issue by rgolea** [angular/material#2977]
Hi there! This is a may be a small bug or just that I don't really know how to use VoiceOver but I'm trying to make sure my website works for accessibility. The thing is that the secondary button doesn't really work with the aria label. I've uploaded a screenshot so you can see.

Sorry for the inconvenience guys. Keep up the good work!

Thanks

P.D.: This is the code I'm using for the seconary click.
```
<md-icon class="md-secondary" ng-click="delete($event, setting)" aria-label="Delete this setting" md-svg-icon="/icons/fi-trash.svg"></md-icon>
```

![screen shot 2015-05-26 at 10 58 09](https://cloud.githubusercontent.com/assets/8843202/7808841/5282fd02-0396-11e5-8e62-8431b1f25db1.png)

**Comment by marcysutton** [comment]
The `aria-label` is being negated with `aria-hidden="true"`, which is dynamically being added for some reason. `md-icon` does some magic to hide the element if there is no `aria-label` present, but it looks like a timing issue with the list controls, which wrap icons with `ng-click` in a more accessible button.

**Comment by ThomasBurleson** [comment]
@marcysutton - `md-icon` with `ng-click` is not supported. Icons are not interactive and should be wrapped in an `md-button` for click features.

**Comment by rgolea** [comment]
Is there a workaround?

**Comment by marcysutton** [comment]
I completely agree, but unless the API changed that is how list controls work. 

The workaround is to put ng-click directly on an md-button element that wraps md-icon, instead of letting mdList transform it. The aria-label can go on either element.

**Comment by EladBezalel** [comment]
icon button is supported for long time now with `md-icon-button` class
https://material.angularjs.org/HEAD/demo/button

**Comment by marcysutton** [comment]
That is for a standalone icon button, I think this was related to list controls. But it's been a while.
